<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PostHelperWindow</class>
 <widget class="QMainWindow" name="PostHelperWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PostHelperWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="usedClassLabel">
      <property name="text">
       <string>Used class: </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="parcelRadioButton">
        <property name="text">
         <string>Parcel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="valueParcelRadioButton">
        <property name="text">
         <string>Parcel with defined value</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="electronicTranferRadioButton">
        <property name="text">
         <string>Electronic transfer</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>From</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="fromNameEdit">
          <property name="placeholderText">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="fromSurnameEdit">
          <property name="placeholderText">
           <string>Surname</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>To</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="toNameEdit">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="toSurnameEdit">
          <property name="placeholderText">
           <string>Surname</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Weight</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="weightEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Value</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="valueEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Sum</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="sumEdit"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="confirmationButton">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Repeat previous of this type</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
